packages/snaps-rpc-methods/src/restricted/getBip44Entropy.ts

```diff
@@ -113,11 +126,11 @@ export function getBip44EntropyImplementation({
     // `args.params` is validated by the decorator, so it's safe to assert here.
     const params = args.params as GetBip44EntropyParams;
 
-    const node = await BIP44CoinTypeNode.fromDerivationPath([
-      await getMnemonic(),
-      `bip32:44'`,
-      `bip32:${params.coinType}'`,
-    ]);
+    const node = await BIP44CoinTypeNode.fromDerivationPath(
+      [await getMnemonic(), `bip32:44'`, `bip32:${params.coinType}'`],
+      'mainnet',
```

Comment:
Should the return value always use `mainnet` here? 🤔 I guess technically the same thing applies to `getBip32Entropy`

Reply:
This matches the previous behaviour. The network is only used for generating extended keys, and on the Snap side you could easily create a node with `testnet` if preferred.

Reply:
Fair enough, we could always add an optional param for this I guess, but no need right now